widget/playground/src/containers/SupportedBlockchains/SupportedBlockchains.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Type } from '../../types';
+
+import { ChainsIcon } from '@rango-dev/ui';
+import React from 'react';
+
+import { MultiSelect } from '../../components/MultiSelect/MultiSelect';
+import { getCategoryNetworks } from '../../helpers';
+import { useConfigStore } from '../../store/config';
+import { useMetaStore } from '../../store/meta';
+
+export function SupportedBlockchains({ type }: { type: Type }) {
+  const {
+    config: { from, to },
+    onChangeBlockChains,
+    onChangeBlockChain,
+    onChangeToken,
+  } = useConfigStore();
+  const {
+    meta: { blockchains },
+  } = useMetaStore();
+
+  const selectedType = type === 'Source' ? from : to;
+
+  const configBlockchains = selectedType?.blockchains;
+
+  const allBlockchain = blockchains.map((chain) => {
+    const { shortName: title, logo, name } = chain;
+    return {
+      title,
+      logo,
+      type: name,
+      networks: getCategoryNetworks([chain]),
+    };
+  });
+
+  const handleBlockchainChange = (items?: string[]) => {
+    // Reset default blockchain and token when the default is not among the selected items.
+    if (
+      selectedType?.blockchain &&
+      items &&
+      !items.includes(selectedType.blockchain)
+    ) {
+      onChangeBlockChain(undefined, type);
+      onChangeToken(undefined, type);
+    }
+    onChangeBlockChains(items, type);
+  };
+
+  return (
+    <MultiSelect
+      label="Supported Blockchains"
+      icon={<ChainsIcon />}
+      type="Blockchains"
+      value={
+        configBlockchains?.length === allBlockchain.length
+          ? undefined
+          : configBlockchains
+      }
+      config={configBlockchains || allBlockchain.map((chain) => chain.type)}
```

Comment:
It was a bit confusing at first because configBlockhains are list of blockchain names and right hand seems to be list of blockchain types. :confused:
I understand it's correct. But why we are using `type` for the name?

Reply:
fixed.